test/test.cfg.example

```diff
@@ -31,6 +31,10 @@ docker-registry-host = localhost:5000
 # WARNING: this will get wiped!  Don't run tests against something that you may want to keep!
 mongodb-database = catalog-test
 
+# Whether to enable ('true') the MongoDB retryWrites parameter or not (anything other than 'true').
+# See https://www.mongodb.com/docs/manual/core/retryable-writes/
+mongodb-retrywrites=false
+
```

Comment:
I don't think this belongs in the test config; if we want to test it we should write tests that write a config and start the service with it set up each way

Reply:
https://github.com/kbase/catalog/pull/146/files#diff-57f4f62dfb5a8954a55eb60ed8210d2c01e82dfe5ac5a8fe5aa0d90ddd8ccfadR43

Ah, but this file uses test.cfg for testing. So, are you suggesting it should load the deploy.cfg file because it has the mongodb-retrywrites parameter, or should we create a fake config to control which parameters we want inside?

Reply:
I'm saying the tests should write a deploy.cfg file. That's probably too much work for this PR though.

Reply:
removed

Reply:
:+1: